app/src/main/java/com/example/reply/ui/ReplyDetailsScreen.kt

```diff
@@ -0,0 +1,276 @@
+                modifier = modifier
+                    .fillMaxWidth()
+                    .padding(
+                        vertical = dimensionResource(R.dimen.detail_button_bar_padding_vertical)
+                    ),
+                horizontalArrangement = Arrangement.spacedBy(
+                    dimensionResource(R.dimen.detail_button_bar_item_spacing)
+                ),
+            ) {
+                ActionButton(
+                    text = stringResource(id = R.string.move_to_inbox),
+                    onButtonClicked = displayToast,
+                    modifier = Modifier.weight(1f)
+                )
+                ActionButton(
+                    text = stringResource(id = R.string.delete),
+                    onButtonClicked = displayToast,
+                    containIrreversibleAction = true,
+                    modifier = Modifier.weight(1f)
+                )
+            }
+        MailboxType.Sent, MailboxType.Inbox ->
+            Row(
+                modifier = modifier
+                    .fillMaxWidth()
+                    .padding(
+                        vertical = dimensionResource(R.dimen.detail_button_bar_padding_vertical)
+                    ),
+                horizontalArrangement = Arrangement.spacedBy(
+                    dimensionResource(R.dimen.detail_button_bar_item_spacing)
+                ),
+            ) {
+                ActionButton(
+                    text = stringResource(id = R.string.reply),
+                    onButtonClicked = displayToast,
+                    modifier = Modifier.weight(1f)
+                )
+                ActionButton(
+                    text = stringResource(id = R.string.reply_all),
+                    onButtonClicked = displayToast,
+                    modifier = Modifier.weight(1f)
+                )
+            }
+    }
+}
+
+@Composable
+private fun DetailsScreenHeader(email: Email, modifier: Modifier = Modifier) {
+    Row(modifier = modifier.fillMaxWidth()) {
+        ReplyProfileImage(
+            drawableResource = email.sender.avatar,
+            description = email.sender.fullName,
+            modifier = Modifier.size(
+                dimensionResource(R.dimen.email_header_profile_size)
+            )
+        )
+        Column(
+            modifier = Modifier
+                .weight(1f)
+                .padding(
+                    horizontal = dimensionResource(R.dimen.email_header_content_padding_horizontal),
+                    vertical = dimensionResource(R.dimen.email_header_content_padding_vertical)
+                ),
+            verticalArrangement = Arrangement.Center
+        ) {
+            Text(
+                text = email.sender.firstName,
+                style = MaterialTheme.typography.labelMedium
+            )
+            Text(
+                text = email.createdAt,
+                style = MaterialTheme.typography.labelMedium,
+                color = MaterialTheme.colorScheme.outline
```

Comment:
i see you are setting a lot of colors in this app, curious why you aren't using the default colors for components